.github/workflows/tests.yml

```diff
@@ -20,12 +20,12 @@ jobs:
       - name: Build
         run: make
 
-      - name: Run tests
-        run: sudo python3 -m unittest -v
-
       - name: Run linters
         run: make lint
 
+      - name: Run tests
+        run: sudo python3 -m unittest -v
```

Comment:
Running linters before test may be a temporary convenience for this, but we shouldn't land this. Clearly functionality is more important than code style? I.e. if a MR fails, that's the more important piece of information than "your line is too long".